members/SMNU26279_811381.yaml

```diff
@@ -0,0 +1,56 @@
+bioguide: SMNU26279_811381
+contact_form:
+  driver: chrome
+  steps:
+    - visit: https://www.senate.mn/members/email-form/1274
+    - fill_in:
+        - name: First Name
+          selector: "#first_name"
+          value: $NAME_FIRST
+          required: true
+        - name: Last Name
+          selector: "#last_name"
+          value: $NAME_LAST
+          required: true
+        - name: email
+          selector: "#email"
+          value: $EMAIL
+          required: true
+        - name: street
+          selector: "#street_address_1"
+          value: $ADDRESS_STREET
+          required: true
+        - name: street2
+          selector: "#street_address_2"
+          value: $ADDRESS_STREET_2
+          required: false
+        - name: phone
+          selector: "#phone"
+          value: $PHONE_HYPHENS
+          required: false
+        - name: city
+          selector: "#city"
+          value: $ADDRESS_CITY
+          required: true
+        - name: zipcode
```

Comment:
Do we also need to include the state dropdown?  I realize our own geo-restricting shouldn't allow out of state, so maybe we normally don't bother

Reply:
Ok, I'm good with leaving it off then.